realsense2_camera/cfg/base_d400_params.py

```diff
@@ -5,8 +5,8 @@
 
 def add_base_params(gen, prefix):
   #             Name                                               Type    Level Description                  Default    Min     Max
-  gen.add(str(prefix) + "depth_gain",                              int_t,    1,  "Gain",                      16,        16,     248)
-  gen.add(str(prefix) + "depth_enable_auto_exposure",              bool_t,   2,  "Enable Auto Exposure",      True)
+  gen.add(str(prefix) + "depth_gain",                              int_t,    1,  "Gain",                      32,        16,     248)
```

Comment:
Is this going to cause problems in Dodgers Stadium?

Reply:
It shouldn't, unless the amount of near IR light in the Dodgers environment is way different than in K1. Only way to find out is to upgrade those cameras and test